easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'PythonPackage'
+
+name = 'MUSCLE3-Python'
+version = '0.5.0'
+
+homepage = 'https://muscle3.readthedocs.io/en/latest/index.html'
+
+description = """MUSCLE3 allows connecting multiple simulation models
+ together into a multiscale simulation. Simulation models can be as
+ simple as a single Python file, or as complex as a combination of
+ multiple separate simulation codes written in C++ or Fortran, and
+ running on an HPC machine."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/multiscale/muscle3/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['abdbad412929d16a303b7e2909d2f1267857bc33514339c1c6f0b89dff2f5b4f']
+
+dependencies = [
```

Comment:
Missing `binutils` build dep as above.

Reply:
@verdurin Changed to use foss toolchain so not needed now.